package.json

```diff
@@ -50,7 +50,7 @@
     "test:chrome-headless": "lerna run test:chrome-headless --scope \"@jupyterlab/test-*\" --concurrency 1 --stream",
     "test:firefox": "lerna run test:firefox --scope \"@jupyterlab/test-*\" --concurrency 1 --stream",
     "test:ie": "lerna run test:ie --scope \"@jupyterlab/test-*\" --concurrency 1 --stream",
-    "tslint": "tslint --fix -c tslint.json '**/*{.ts,.tsx}'",
+    "tslint": "tslint --fix -c tslint.json --project tsconfigbase.json '**/*{.ts,.tsx}'",
```

Comment:
In order for tslint to reason about types, we need to include the `tsconfig` file.